scalabel/label/from_waymo.py

```diff
@@ -0,0 +1,369 @@
+"""Convert Waymo Open format dataset to Scalabel."""
+import argparse
+import glob
+import math
+import os
+from functools import partial
+from typing import List, Tuple
+
+import numpy as np
+
+try:
+    from simple_waymo_open_dataset_reader import (
+        WaymoDataFileReader,
+        dataset_pb2,
+        label_pb2,
+        utils,
+    )
+
+    # pylint: disable=no-member
+    classes_type2name = {
+        label_pb2.Label.Type.TYPE_VEHICLE: "vehicle",
+        label_pb2.Label.Type.TYPE_PEDESTRIAN: "pedestrian",
+        label_pb2.Label.Type.TYPE_CYCLIST: "cyclist",
+        label_pb2.Label.Type.TYPE_SIGN: "sign",
+    }
+    # pylint: enable=no-member
+except ImportError:
+    WaymoDataFileReader = None
+
+from ..common.parallel import pmap
+from .io import save
+from .typing import (
+    Box2D,
+    Box3D,
+    Extrinsics,
+    Frame,
+    ImageSize,
+    Intrinsics,
+    Label,
+)
+from .utils import (
+    get_extrinsics_from_matrix,
+    get_matrix_from_extrinsics,
+    get_matrix_from_intrinsics,
+)
+
+cameras_id2name = {
+    1: "FRONT",
+    2: "FRONT_LEFT",
+    3: "FRONT_RIGHT",
+    4: "SIDE_LEFT",
+    5: "SIDE_RIGHT",
+}
+
+
+def parse_arguments() -> argparse.Namespace:
+    """Parse the arguments."""
+    parser = argparse.ArgumentParser(description="waymo to scalabel")
+    parser.add_argument(
+        "--input",
+        "-i",
+        help="path to Waymo data (tfrecords).",
+    )
+    parser.add_argument(
+        "--output",
+        "-o",
+        default="./",
+        help="Output path for Scalabel format annotations.",
```

Comment:
It is better to specify this is the output file path to be symmetric with the input file path.

Reply:
okay i removed the default parameter